src/systems/a320_systems_wasm/src/autobrakes.rs

```diff
@@ -0,0 +1,39 @@
+use std::error::Error;
+use std::time::Duration;
+use systems_wasm::aspects::{EventToVariableMapping, EventToVariableOptions, MsfsAspectBuilder};
+use systems_wasm::Variable;
```

Comment:
We might want to group imports from the same crate, but this is not important to me. This is also relevant for the other files.

Reply:
I agree. What I've found is that CLion and vscode handle these things differently and all have their quirks. For example, if I manually group these in CLion then any additional import added through the IDE within the `::aspects::` bit would go to a new line. Probably because it doesn't understand nested imports to that many levels, at least not for formatting/code completion purposes.

For now I guess we'll just have to accept this is how it is. I'll be sure to keep an eye out for improvements in this area. If at some point the editors work fine, we can create a PR that makes all imports throughout the project nicer.